test/Microsoft.Extensions.Internal.Test/Microsoft.Extensions.Internal.Test.csproj

```diff
@@ -5,6 +5,7 @@
   <PropertyGroup>
     <TargetFrameworks>netcoreapp2.0;net461</TargetFrameworks>
     <TargetFrameworks Condition=" '$(OS)' != 'Windows_NT' ">netcoreapp2.0</TargetFrameworks>
+    <DebugType>portable</DebugType>
```

Comment:
It's xunit shenanigans (https://github.com/xunit/xunit/blob/master/src/xunit.core/build/xunit.core.props#L14) and not the SDK. Maybe a bug in it. Either way, forcing this to portable ensures we get the stack trace correctly both on desktop and coreclr. In the absence of this, coreclr is missing the stacktrace for the test file.